packages/demo/backend/src/services/wallet.ts

```diff
@@ -204,5 +205,5 @@ export async function sendTokens(
     throw new Error('Wallet not found')
   }
 
-  return wallet.sendTokens(amount, 'usdc', recipientAddress)
+  return wallet.sendTokens(amount, USDC, baseSepolia.id, recipientAddress)
```

Comment:
I think we can move past hard coding these, Where do we set the demo vault network?